docs/introduction.rst

```diff
@@ -3,49 +3,86 @@ An Introduction to ndcube
+Take the example of an image of the Sun taken from Earth.
+The world axis of helioprojective longitude is dependent on helioprojective latitude, i.e. there is not one pixel axis for longitude and another for latitude.
+Both world axes are associated with both pixel axes.
+In an APE-14-compliant WCS object, the mapping between pixel and world axes is described by the `~astropy.wcs.WCS.axis_correlation_matrix`.
+
+Due to unfortunate convention, WCS orders its pixel axes in the inverse order to numpy.
+Therefore we use the term "array axes" to refer to pixel axes which have been reversed to reflect the axis order of the numpy data array.
+Take for example a numpy array with three dimensions.
+Since the array axes are simply the reverse of the pixel axes, the first axis of the array corresponds to the 3rd pixel axis.
+And the 2nd array axis corresponds to the 2nd pixel axis.
+If the array had four axes, the first array axis would correspond to the fourth pixel axis and the second array axis would correspond to the third pixel axis and so on.
+
+In ndcube, inputs and outputs are given in either array axis order or world axis order, depending on the types of information.
+Throughout these docs and in the docstrings we will endeavor to highlight which order is relevant.
+However a good rule of thumb is that if you are using a sequence of coordinate objects to describe locations in the data cube -- for example in the input of `ndcube.NDCube.crop` or the output of `ndcube.NDCube.axis_world_coords` -- they should be in world axis order.
+In almost all other cases, array axis order is used.
 
 Why ndcube?
------------
-The most similar project to ndcube is xarray and it’s worth addressing why ndcube exists in its own right, rather than devoting time and effort to building tools around xarray.
-The fundamental reason to opt for ndcube is a requirement to harness the astronomy-specific World Coordinate System (WCS) tooling provided by packages like Astropy (which now also supports gWCS).
-The data model of xarray centered on the requirements and conventions of the geosciences, which while being very similar to that of astronomy in conception, is sufficently different in construction to cause much friction.
-Utilize the astropy WCS infrastructure enables us to directly read the most common file formats in astronomy (FITS), although the FITS WCS data model is also commonly used outside of FITS files.
+===========
+It's worth addressing the role ndcube plays within the scientific Python ecosystem and why it exists separately from its most similar package, xarray.
+The fundamental reason to opt for ndcube is to harness the astronomy-specific World Coordinate System (WCS).
+The data model of xarray centers on the requirements and conventions of the geosciences.
+Although very similar to those of astronomy in conception, they are sufficently different in construction to cause significant friction.
```

Comment:
I dont know why, but friction here sounds like theres been an argument between geosciences and astronomy :P - would `challenges` work better?

Reply:
Ha!  Oh I see why you made a comment on this before now.  I thought friction described it better because "challenge" makes me think of a more philosophical problem that hasn't been solved.  But now I see where you're coming from I'm not sure.